guidelines/enhancement_template.md

```diff
@@ -272,6 +272,40 @@ to implement the design.  For instance,
  > 1. This requires exposing previously private resources which contain sensitive
   information.  Can we do this?
 
+## Security
```

Comment:
There is an existing:

> How will security be reviewed and by whom?

Up under the *Risks and Mitigations* section.  Did we want to drop the security-related wording there now that we're growing a dedicated section here?

Reply:
Yep, that seems reasonable, I think that is kind of covered now by this section